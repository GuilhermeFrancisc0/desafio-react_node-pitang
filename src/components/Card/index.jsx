import React from 'react';
import { Card } from 'react-bootstrap';

export default function index({ title, children }) {
  return (
    <Card>
      <Card.Header>
        <Card.Title>{title}</Card.Title>
      </Card.Header>
      <Card.Body>
        {children}
      </Card.Body>
    </Card>
  );
}
