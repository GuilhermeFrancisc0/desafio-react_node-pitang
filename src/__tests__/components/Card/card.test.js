/* eslint-disable no-undef */
import React from 'react';

import { render } from '@testing-library/react';
import Card from '../../../components/Card';

test('Card renders', () => {
  const { asFragment } = render(<Card />);

  expect(asFragment()).toMatchSnapshot();
});
