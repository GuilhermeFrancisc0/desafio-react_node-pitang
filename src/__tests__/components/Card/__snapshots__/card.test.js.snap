// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <div
      class="card-header"
    >
      <div
        class="card-title h5"
      />
    </div>
    <div
      class="card-body"
    />
  </div>
</DocumentFragment>
`;

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <div
      class="card-header"
    >
      <div
        class="card-title h5"
      />
    </div>
    <div
      class="card-body"
    />
  </div>
</DocumentFragment>
`;
